zeppelin-zengine/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -506,6 +510,11 @@ public String getWebsocketMaxTextMessageSize() {
     return getString(ConfVars.ZEPPELIN_WEBSOCKET_MAX_TEXT_MESSAGE_SIZE);
   }
 
+  public Set<String> getDefaultOwner() {
+    String values = getString(ConfVars.ZEPPELIN_DEFAULT_OWNER);
+    return new HashSet<>(Arrays.asList(COMMA_DELIMITER.split(values)));
```

Comment:
yes, I added new unit test for checking if current user is set when ZEPPELIN_DEFAULT_OWNER is empty.